### ISSUE TITLE
Converting lime explainer output to json

### ISSUE THREAD

**Issue by vaisakhcherrylabs** [prrao87/fine-grained-sentiment-app#3]
is it possible to convert the lime explainer output to json? or suggest any method to predict the final output without using lime framework

**Comment by prrao87** [comment]
You can return a list or map from the LIME explainer [as per these methods](https://lime-ml.readthedocs.io/en/latest/lime.html#lime.explanation.Explanation) shown in the docs. I don't know of a straightforward way to output to JSON, but see [this idea on how to convert it to JSON via pandas](https://github.com/marcotcr/lime/issues/96).

I haven't really worked with other explainers. but may be try and compare ELI5, LIME and SHAP to see how they differ from each other!

**Comment by vaisakhcherrylabs** [comment]
>     exp = explainer(method,
>                         path_to_file=METHODS[method]['file'],
>                         text=text,
>                         num_samples=int(n_samples))
> 
>     explain = exp.as_map()

This change can able to convert it to map. but the output is not json serializable
